components/dashboard/src/projects/NewProject.tsx

```diff
@@ -312,7 +313,7 @@ export default function NewProject() {
 
     return (<div className="flex flex-col w-96 mt-24 mx-auto items-center">
         <h1>New Project</h1>
-        <p className="text-gray-500 text-center text-base">Projects allow you to set up and acess Prebuilds.</p>
+        <p className="text-gray-500 text-center text-base">Select a git repository.</p>
```

Comment:
**issue**: This is missing the stepper component as seen in the specs in https://github.com/gitpod-io/gitpod/issues/4948. ● ○ ○